packages/create-turbo/templates/_shared_ts/package.json

```diff
@@ -13,6 +13,7 @@
     "format": "prettier --write \"**/*.{ts,tsx,md}\""
   },
   "devDependencies": {
+    "eslint-config-custom": "workspace:*",
```

Comment:
This is a pnpm-specific microformat, and it should actually come in from the pnpm directory.

The other two changes should get this to work.

Reply:
But, it should still be here, just as "eslint-config-custom": "*".

Reply:
Ah great call

Reply:
Also here:
- https://github.com/vercel/turborepo/blob/main/packages/create-turbo/templates/npm/package.json#L15
- https://github.com/vercel/turborepo/blob/main/packages/create-turbo/templates/yarn/package.json#L15

`pnpm` was the *closest* to working.

Reply:
There was actually a bigger issue here, we're not merging dependencies between the package specific package.json, and the root package.json.